pkg/destroy/azure/azure.go

```diff
@@ -100,26 +101,85 @@ func New(logger logrus.FieldLogger, metadata *types.ClusterMetadata) (providers.
 }
 
 // Run is the entrypoint to start the uninstall process.
-func (o *ClusterUninstaller) Run() error {
+func (o *ClusterUninstaller) Run() (err error) {
```

Comment:
I personally try to avoid named returns because it allows `return` in code that makes it unclear what is being returned.

Reply:
Fixed.